source/nordic-sdk/components/libraries/bootloader_dfu/dfu_app_handler.c

```diff
@@ -30,29 +30,28 @@
  *
  */
 
-#include <string.h>
 #include "dfu_app_handler.h"
+#include <string.h>
 #include "bootloader_util.h"
 #include "nrf.h"
 #include "nrf_sdm.h"
+#include "ble_gatt.h"
 #include "ble_gatts.h"
 #include "app_error.h"
 #include "dfu_ble_svc.h"
 #include "device_manager.h"
+#include "nrf_delay.h"
 
-#define IRQ_ENABLED            0x01                                            /**< Field identifying if an interrupt is enabled. */
-#define MAX_NUMBER_INTERRUPTS  32                                              /**< Maximum number of interrupts available. */
+#define IRQ_ENABLED            0x01                                     /**< Field that identifies if an interrupt is enabled. */
+#define MAX_NUMBER_INTERRUPTS  32                                       /**< Maximum number of interrupts available. */
 
-static void                    dfu_app_reset_prepare(void);                    /**< Forward declare of default reset handler. */
-static dfu_app_reset_prepare_t m_reset_prepare = dfu_app_reset_prepare;        /**< Callback function to application to prepare for system reset. Allows application to cleanup of service and memory prior to reset. */
-#if NEEDED
-static dfu_ble_peer_data_t     m_peer_data;                                    /**< Peer data to be used for data exchange when reseting into DFU mode. */
-static dm_handle_t             m_dm_handle;                                    /**< Device Manager handle with instance id's of current BLE connection. */
```

Comment:
just marking this spot for my own reference. I don't recall why I added the `#if NEEDED` here. Perhaps the original reason has been lost as we upgraded to newer SDKs.